dist/angular-patternfly.js

```diff
@@ -1670,6 +1683,7 @@ angular.module('patternfly.charts').directive('pfHeatmap', ["$compile", "$window
        }
 
        $scope.data = {
+         dataAvailable: true,
```

Comment:
@simon3z this is how trend-chart and heat-maps works, so all the cards will behave the same.
If we do it this way, it will just work in manageiq without changing anything :-).
Doing it by checking the data will need to change the overview in manageIQ.